src/components/settings-row/settings-row.component.js

```diff
@@ -58,6 +60,10 @@ SettingsRow.propTypes = {
   className: PropTypes.string,
   /**  A title for this group of settings. */
   title: PropTypes.string,
+  /** Defines the HTML heading element which is applied to the `title` within the component. */
+  headingType: PropTypes.arrayOf(
```

Comment:
why are we making this an array in PropTypes? It should just be `headingType: PropTypes.oneOf(["h1", "h2", "h3", "h4", "h5", "h6"])` or consumers will see PropTypes errors everywhere when using this component.

Please let me know if I'm missing something obvious!